src/vehicle.cpp

```diff
@@ -7727,9 +7755,9 @@ int vehicle::part_count( bool no_fake ) const
     return no_fake ? std::count_if( parts.begin(), parts.end(), []( const vehicle_part & vp ) {
         return !vp.is_fake;
     } ) : static_cast<int>( parts.size() );
-}
+    }
```

Comment:
Are these spaces causing the astyle test to fail?

Reply:
Most likely

Reply:
Good catch, I think AStyle got confused by the format of that method. Thanks for taking a look!